kieserver/image.yaml

```diff
@@ -142,6 +142,9 @@ envs:
   - name: "KIE_SERVER_DISABLE_KC_VERIFICATION"
     example: "true"
     description: "Disable Maven KIE Jar verification. It is recommended to test the kjar manually before disabling this verification."
+  - name: "KIE_SERVER_JBPM_CLUSTER"
```

Comment:
please cherry pick changes to RHPAM Kie Server image as well

Reply:
ok